tests/test_minimal/test_tools/test_nwb_helpers.py

```diff
@@ -25,12 +26,10 @@ def test_get_module(self):
         assert mod_2.description == description_2
         self.assertWarns(UserWarning, get_module, **dict(nwbfile=nwbfile, name=name_1, description=description_2))
 
-    def test_make_nwbfile_from_metadata(self):
-        with self.assertRaisesWith(
-            exc_type=AssertionError,
-            exc_msg=(
-                "'session_start_time' was not found in metadata['NWBFile']! Please add the correct start time of the "
-                "session in ISO8601 format (%Y-%m-%dT%H:%M:%S) to this key of the metadata."
-            ),
-        ):
+    def test_make_nwbfile_from_metadata_empty(self):
+        with self.assertRaises(ValidationError):
             make_nwbfile_from_metadata(metadata=dict())
+
```

Comment:
Thanks, these tests are good at showing what kinds of basic `metadata` structures are now prohibited from being used to make a file